### PR TITLE
 Create landing page with Search functionality

### PR DESCRIPTION
**Pull request by ahegazy**
A Vue.js App that helps you find books, retrieve book's ratings, author and similar books .. It fetches search results from Goodreads and Google books.

As a user I want to search for books by name, author, ISBN, so that I can find information about any book.

### CODE REVIEW

File: BookFinder/build/build.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+'use strict'
+require('./check-versions')()
+
+process.env.NODE_ENV = 'production'
```
Comment (adeen-s) [inline BookFinder/build/build.js]: The NODE_ENV should be currently set to 'development' to provide better logs and facilitate development.

File: BookFinder/src/router/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import HelloWorld from '@/components/HelloWorld'
+
+Vue.use(Router)
+
+export default new Router({
+  routes: [
+    {
+      path: '/',
+      name: 'test',
+      component: HelloWorld
```
Comment (adeen-s) [inline BookFinder/src/router/index.js]: Remove the HelloWorld route and add custom routes for our app.

**Review by adeen-s — CHANGES_REQUESTED** [review]
Great work @ahegazy Absolutely love it.
There are a few comments which I have mentioned. Fix those and then we can get to merging this.

File: BookFinder/src/components/book.vue [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+                break;
+            default: 
+                this.$router.push('/');
+           }
+
+
+        },
+        GetbookGoodread: function (id){
+            var ResultsObj = {};
+            this.$http.get('https://cors-anywhere.herokuapp.com/https://www.goodreads.com/book/show/'+id+'.xml?key=whM4DZOLcr7BsKwZs1Gclw')
```
Comment (adeen-s) [inline BookFinder/src/components/book.vue]: You have exposed your API key to the public. Make sure you remove it from here and also regenerate your key.

**Review by adeen-s — CHANGES_REQUESTED** [review]

File: BookFinder/src/components/book.vue (hunk 3 above)
Comment (danivijay) [reply BookFinder/src/components/book.vue]: I think we can make use of dotenv here https://www.npmjs.com/package/dotenv

**Review by danivijay — COMMENTED** [review]
Vuex can make things better, but it can be done later.

**Comment by adeen-s** [comment]
LGTM

**Review by adeen-s — APPROVED** [review]

**Pull request merged**